src/RA_Dlg_AchEditor.cpp

```diff
@@ -300,7 +304,8 @@ LRESULT CALLBACK DropDownProc(HWND hwnd, UINT nMsg, WPARAM wParam, LPARAM lParam
             ZeroMemory(&lvDispinfo, sizeof(LV_DISPINFO));
             lvDispinfo.hdr.hwndFrom = hwnd;
             lvDispinfo.hdr.idFrom = GetDlgCtrlID(hwnd);
-            lvDispinfo.hdr.code = LVN_ENDLABELEDIT;
+#pragma warning(suppress: 26454)
+            GSL_SUPPRESS(io.5) lvDispinfo.hdr.code = LVN_ENDLABELEDIT;
```

Comment:
`LVN_ENDLABELEDIT` is a negative `UINT`, I have no idea what the design behind it means but just suppressed it. It does show up as` MAX_UINT` but almost everything in the file it came from is as well. The API probably does some weird stuff with it or uses the name instead of the value.